source/views/help/wifi.js

```diff
@@ -80,15 +81,45 @@ export class ReportWifiProblemView extends React.Component<Props, State> {
 
   render() {
     return (
-      <ScrollView>
-        <Card footer={this.state.status} header="Report a WiFi Problem!">
-          <Button
-            disabled={this.state.status !== ''}
-            onPress={this.start}
-            title="Report"
-          />
+      <View>
+        <Card footer={this.state.status}>
+          <Container>
+            <Title selectable={true}>Report a Wi-Fi Problem</Title>
+            <Description selectable={true}>
+              If you are having an issue connecting to any of the St. Olaf
+              College Wi-Fi networks, please tap the button below.
+            </Description>
+            <Description selectable={true}>
+              We will record your current location and what device you are
```

Comment:
>and what device you are using

could be something more like

>and some general information about the device you are using

Reply:
Should we change that in the release notes, too?